circle.yml

```diff
@@ -0,0 +1,17 @@
+machine:
+  python:
+    version: 3.4.3
+
+dependencies:
+  override:
+    - pip install tox tox-pyenv
+    - pyenv local 2.7.10 3.4.3
+    - pip install -U pip setuptools
+    - pip install -U -e ".[dev]"
+
+test:
+  override:
+    - make test
+    - tox
+  post:
+    - coverage html -d $CIRCLE_ARTIFACTS
```

Comment:
Running both is probably not necessary. I should save the environment and remove the circle config .)